recipes/fenics-oasis/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "fenics-oasis" %}
+{% set version = "2019.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mikaem/Oasis/archive/{{ version }}.tar.gz
+  sha256: 4456d5f042292e30bffda0487b57e282fe6e776cb6cdc832140072983b2c9d8d
+
+build:
+  entry_points:
+    - oasis=oasis.run_oasis:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - fenics
+    - numpy
+    - matplotlib-base
+    - procps-ng  # [linux]
```

Comment:
This one is also "arch'ed", is it optional?

Reply:
@ocefpaf Yes, I have removed `procps-ng` now since it is only used for optional memory usage information and is not strictly needed.